cypress/integration/admin.js

```diff
@@ -0,0 +1,41 @@
+import LoginPage from '../objects/loginPage';
+import TaskPage from '../objects/TaskPage';
+
+const loginPage = new LoginPage();
+const taskPage = new TaskPage();
+
+describe('Tests related to Admin role', function() {
+  beforeEach(function() {
+    cy.loginAs('Admin');
+    loginPage.visit();
+  });
+
+  it('Admin should be able to create new task', function() {
+    taskPage.visit();
+    taskPage.getCreateTaskButton().click();
+
+    cy.url().should('include', 'tasks/create');
+    taskPage
+      .getTaskNameInput()
+      .type('Admin task ')
```

Comment:
please make this random in order to run it multiple times

Reply:
Random generation of task name was implemented.